microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Analyze selector performance during Recalculate Style events
+description: Use the Selector Stats feature in the Performance tool to understand which of your CSS selectors are taking up the most time during Recalculate Style events.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/09/2023
+---
+# Analyze selector performance during Recalculate Style events
+
+The **Performance** tool highlights long tasks with red triangles to indicate work on the main thread that takes too long and has slow performance.  
+
+![Long Task indicators for events on the main thread with slow performance.](../media/evaluate-performance-performance-animation-frame-fired.msft.png)
```

Comment:
```suggestion
![Long Task indicators for events on the main thread with slow performance](../media/evaluate-performance-performance-animation-frame-fired.msft.png)
```
moving away from period, this might be noted in wiki > Edge Dev Style Guide.  for simple-as-possible Markdown